src/electron/flux/store/left-nav-store.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { BaseStoreImpl } from 'background/stores/base-store-impl';
+import { StoreNames } from 'common/stores/store-names';
+import { LeftNavStoreData } from 'electron/flux/types/left-nav-store-data';
+import { LeftNavActions } from 'electron/flux/action/left-nav-actions';
+import { LeftNavItem } from 'electron/types/left-nav-item';
+import { LeftNavItemKey } from 'electron/types/left-nav-item-key';
+
+export class LeftNavStore extends BaseStoreImpl<LeftNavStoreData> {
+    constructor(private readonly actions: LeftNavActions) {
+        super(StoreNames.LeftNavStore);
+    }
+
+    public getDefaultState(): LeftNavStoreData {
+        return {
+            selectedKey: 'overview',
```

Comment:
We should change this to automated checks since that is the default for unified.